### ISSUE TITLE
Custom keybinds are not working?

### ISSUE THREAD

**Issue by Iveiyh** [valignatev/hitboxer#44]
Edited the socd_v008.conf file to the following values
51
44
5A
53
45
0

which should correspond to Azerty keybinds, yet when opening the program and clicking either of the three options it just doesn't work? Or am I just plain stupid and doing it wrong, could be either.

Thanks in advance!

**Comment by Iveiyh** [comment]
Actually, this might be because the config file is called socd_v008.conf and not just socd.conf? Cause a previous version with these set changes work fine, I'll have to try that out in a bit.

**Comment by valignatev** [comment]
Hey, yeah I changed the config file name (added the version) because upcoming releases will change the format as well. Hope it's not too confusing lol.

**Comment by Iveiyh** [comment]
Yeah, but the old version the rebinding works fine, in the newer one I can change whatever in the config but it just doesn't work.

**Comment by valignatev** [comment]
Doesn't work as in - it still socds WASD, or the program doesn't work at all? If it's the first case - make sure you restart the program after changing the config, and if the second - give it a try with admin privileges.

**Comment by Iveiyh** [comment]
It still SOCDS using the keys WASD, the program does work for example when holding D and pressing A, but not when holding Z and pressing S, I had to switch to Qwerty layout for this, but that made sure the program was still working.

**Comment by valignatev** [comment]
Yeah I don't know then, I've only tested it with Russian kb layout and it worked. I'll try to install AZERTY and see what's different about it. I guess you could use the previous version for now :(

**Comment by valignatev** [comment]
New release, maybe this is fixed
